cmd/utils/flags.go

```diff
@@ -537,6 +537,12 @@ var (
 		Usage: "Minimum POW accepted",
 		Value: whisper.DefaultMinimumPoW,
 	}
+
+	//Sharding Settings
+	ValidatorSetFlag = cli.BoolFlag{
+		Name:  "validator",
+		Usage: "Join validator set",
```

Comment:
Please elaborate on the usage. The user needs to understand the consequences of this flag.